### PR TITLE
Create a SAT 2.4.11 release that includes cray-sat 3.19.2

### PR DESCRIPTION
**Pull request by haasken-hpe**
## Summary and Scope

Create SAT 2.4.11 release. The most notable changes are:

* Update the cray-sat container image to 3.19.2 to include `sat bootprep` updates to handle the name change of the hpc-shasta-software-recipe to hpc-csm-software-recipe.
* Adds Snyk scanning.
* Adds license-checker workflow on this repository rather than at the organization level. 
* Includes a new 3.3.0 version of cfs-config-util that is used by `update-mgmt-ncn-cfs-config.sh`. We don't use the new functionality, but it has been adequately tested and is low risk to pull in.

## Issues and Related PRs

* Part of [CRAYSAT-1559](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1559)
* Required for [CRAYSAT-1552](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1552)
* Resolves [CRAYSAT-1399](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1399)

### Tested on:

  * surtur

### Test description:

The release distribution built by Jenkins from this branch will be downloaded from artifactory and installed on surtur as a sanity check.

## Risks and Mitigations

All low risk changes.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -38,39 +38,10 @@ set -ex
 ROOTDIR="$(dirname "${BASH_SOURCE[0]}")"
 source "${ROOTDIR}/vendor/github.hpe.com/hpe/hpc-shastarelm-release/lib/release.sh"
 
-# Set to "unstable" for artifacts from main branch, "stable" for artifacts from
```
Comment (haasken-hpe) [inline release.sh:41]: Ooops. Bad cherry-pick here. Fixing.

Comment (haasken-hpe) [reply release.sh:41]: I re-did this cherry-pick and made sure to include the call to the new `render_templates.sh` script.

**Comment by haasken-hpe** [comment]
> 

Correct. And I also noticed that commit message issue and fixed it up.

**Comment by haasken-hpe** [comment]
> It looks correct to me except for the cherry-picking issue (I think you just forgot to put in the call to `render_templates.sh`, right?).
> 
> Also, the header of the commit message on the most recent commit should say 2.4.11 rather than 2.4.10, I think.

Correct. And I also noticed that commit message issue and fixed it up.

**Review by jack-stanek-hpe — APPROVED** [review]

**Comment by ekamin-hpe** [comment]
I think you should be able to fix the builds now by re-applying the commit(s) that did the `git vendor update release`.

**Comment by haasken-hpe** [comment]
Install test output here: https://gist.github.hpe.com/ryan-haasken/dda70f18ee038fe28955945704689bf1

Note that the management NCN components were in a failed state in CFS, so I had to follow the procedure to apply the CFS configuration manually from our documentation, which worked smoothly.

**Pull request merged**